storage/migrations/23_evm_nfts.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE chain.evm_nfts
+    ADD COLUMN metadata JSONB;
```

Comment:
using a json type to be nice to people who want to do things in the db. but beware there are no indices. downside is that both go and postgresql will parse the JSON code. if they ever have any difference in opinion on what's valid JSON, then someone could break our indexer.